testing/trino-product-tests/src/main/java/io/trino/tests/product/hudi/TestHudiSparkCompatibility.java

```diff
@@ -320,6 +320,44 @@ public void testTimelineTableRedirect()
         }
     }
 
+    @Test(groups = {HUDI, PROFILE_SPECIFIC_TESTS})
+    public void testReadCopyOnWriteTableWithReplaceCommits()
+    {
+        String tableName = "test_hudi_cow_replace_commits_select_" + randomNameSuffix();
+
+        onHudi().executeQuery(format(
+                """
+                        CREATE TABLE default.%s (
+                          id bigint,
+                          name string,
+                          price int,
```

Comment:
`price` column looks unused. Can we remove it?

Reply:
Yes, I have removed it.